docs/whats-new/dotnet-aspire-9.1.md

```diff
@@ -127,6 +127,10 @@ The `PublishAsDockerfile()` feature was introduced for all projects and executab
 
 In 9.1, we addressed a persistent issue where Docker networks created by .NET Aspire would remain active even after the application was stopped. This bug, tracked in [issue #6504](https://github.com/dotnet/aspire/issues/6504), is resolved. Now, Docker networks are properly cleaned up, ensuring a more efficient and tidy development environment.
 
+### GitHub Codespaces support
```

Comment:
I added a section with links to the new articles already, see:

- https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-aspire-9.1?branch=pr-en-us-2663#improved-onboarding-experience

I'm not certain we need this section, unless you really want to keep it?